docs/site/Debugging-with-VSC.md

```diff
@@ -0,0 +1,26 @@
+---
+lang: en
+title: 'Debugging with VS Code'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Debugging-with-vsc.html
+---
+
+LoopBack team develops with editor Visual Studio Code in which we defined some
+tasks to help people debug their projects.
+
+### Debugging Packages
+
+The monorepo `loopback-next` contains a folder `.vscode` at root level for
+debugging the test files. When start debugging, it will stop at any breakpoint
+triggered by the tests. There are two tasks defined now:
+
+- Run Mocha tests: Debug all tests inside the `/packages` folder.
+- Debug Current Test File: Only debug the tests inside the current open file.
+
```

Comment:
Maybe we can mention Mocha `describe.only` and `it.only` tricks?

Mocha also has `-g` option to filter matching tests to be run.

Reply:
Sounds good, added.